code/__HELPERS/unsorted.dm

```diff
@@ -1391,3 +1391,22 @@ proc/pick_closest_path(value)
 #define RANDOM_COLOUR (rgb(rand(0,255),rand(0,255),rand(0,255)))
 
 #define QDEL_IN(item, time) addtimer(GLOBAL_PROC, "qdel", time, FALSE, item)
+
+/proc/check_for_cleanbot_bug()
+	var/static/admins_warned //bet you didn't know you could do this!
```

Comment:
The original use of `static` before OOP came around (such as in C before c++ added objects) was for procs

it does the same thing, but for that proc. You can kinda think of it as a global var that only exists in that proc.

I do it because global vars add compile time overhead since it has to search thru the global var list by name every time it resolves a global var since they (like proc vars) compile into an index.